### DISCUSSION TITLE
Github <> Slack integration

### DISCUSSION THREAD

**Discussion by lukerQuant** [hyperledger/fabric#2399] [General]
Hi,

Can you add the slack github integration to this repo (https://slack.github.com/), so that changes can be easily tracked in (other companies) slack.

I understand that email notifications can be used, but in my opinion slack notifications provides easier team organisation.

**Comment by guoger** [comment]
not sure how this could be accomplished, is there a particular github app to be installed?

**Reply by lukerQuant** [reply]
Hi there,

It is this app: https://github.com/apps/slack/installations/new 

It helps with other DLT githubs. For instance, typing "/github subscribe corda/corda" in a slack channel would feed in github alerts for corda repo updates

**Reply by guoger** [reply]
i see, maybe this is something that @ryjones could help with?

**Reply by ryjones** [reply]
It was installed for two repos; I updated it to be on all repos

**Comment by lukerQuant** [comment]
Great. I can confirm subscriptions are now allowed into slack for Fabric and Besu via the slack commands "/github subscribe hyperledger/fabric" and "/github subscribe hyperledger/besu" respectively. Many thanks, this will be really useful!
